spec/features/user_can_submit_posts_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-RSpec.feature "Timeline", type: :feature do
```

Comment:
Don't think we need to change this file as it's nothing to do with the logout function